src/MercadoPago/Client/Order/OrderClient.php

```diff
@@ -36,4 +37,21 @@ public function create(array $request, ?RequestOptions $request_options = null):
         $result->setResponse($response);
         return $result;
     }
+
+    /**
+     * Method responsible for obtaining Order by ID
+     *
+     * @param string $order_id Order ID.
+     * @param \MercadoPago\Client\Common\RequestOptions request options to be sent.
+     * @return \MercadoPago\Resources\Order Order obtained
+     * @throws \MercadoPago\Exceptions\MPApiException an error if the request fails.
+     * * @throws \Exception an error if the request fails.
+     */
+    public function get(string  $id, ?RequestOptions $request_options = null): Order
+    {
+        $response = parent::send(sprintf(self::URL_WITH_ID, $id), HttpMethod::GET, null, null, $request_options);
```

Comment:
Faltou renomear para `$order_id` aqui.
```suggestion
    public function get(string $order_id, ?RequestOptions $request_options = null): Order
    {
        $response = parent::send(sprintf(self::URL_WITH_ID, $order_id), HttpMethod::GET, null, null, $request_options);
```